src/fido2_commands.cc

```diff
@@ -815,6 +834,199 @@ absl::variant<cbor::Value, Status> ResetPositiveTest(DeviceInterface* device) {
   return cbor::Value();
 }
 
+absl::variant<cbor::Value, Status>
+AuthenticatorCredentialManagementPositiveTest(DeviceInterface* device,
+                                              DeviceTracker* device_tracker,
```

Comment:
Unrelated to this particular PR but was there a reason to take pointers instead of references? We don't check if they are null.

Reply:
Reason is old style recommendation. Most pointers throughout could become references.

Reply:
Yes indeed, the recommendation has recently changed.